guides/installation/requirements.md

```diff
@@ -49,35 +51,122 @@ You can use these commands to check your actual environment:
   * `ext-xml`
   * `ext-zip`
   * `ext-zlib`
-* Composer recommended version: 2.0 or higher
+* Composer recommended version: 2.2 or higher
+
+<Tabs>
+
+<Tab title="Ubuntu">
+
+Add a new software repository to your system to have the latest PHP version.
+
+```bash
+sudo add-apt-repository ppa:ondrej/php
+
+sudo apt-get install -y php8.3-fpm php8.3-mysql php8.3-curl php8.3-gd php8.3-xml php8.3-zip php8.3-opcache php8.3-mbstring php8.3-intl php8.3-cli
+
+sudo wget https://getcomposer.org/download/latest-stable/composer.phar -O /usr/local/bin/composer
+sudo chmod +x /usr/local/bin/composer
+```
+
+</Tab>
+
+<Tab title="Debian">
+
+Add a new software repository to your system to have the latest PHP version:
+
+```bash
+curl https://packages.sury.org/php/README.txt | bash
+
+sudo apt-get install -y php8.3-fpm php8.3-mysql php8.3-curl php8.3-gd php8.3-xml php8.3-zip php8.3-opcache php8.3-mbstring php8.3-intl php8.3-cli
+
+sudo wget https://getcomposer.org/download/latest-stable/composer.phar -O /usr/local/bin/composer
+sudo chmod +x /usr/local/bin/composer
+```
+
+</Tab>
+
+<Tab title="macOS">
+
+The easiest way is to use [Homebrew](https://brew.sh/):
+
+```bash
+brew install php@8.3
```

Comment:
will brew automatically install composer as well? in the other setups you described how to install composer separately